### DISCUSSION TITLE
Issue with Displaying Image in Next.js Application Using Image Component

### DISCUSSION THREAD

**Discussion by morganm94** [vercel/next.js#63695] [Help] [answered]
### Summary

Hello fellow developers,

I've encountered an issue while integrating the Next.js Image component into my application. Despite confirming that the image URL (imageURL) is valid and accessible, the image fails to render on the webpage. Below is the snippet of the code I've used:

```
import Image from "next/image";

<Image
  src={imageURL}
  width={48}
  height={48}
  alt="logo"
/>
```

I'm seeking insights or suggestions on what might be causing this issue. Could it be related to Next.js image optimization features or a potential misconfiguration in my Next.js setup? Any advice or troubleshooting tips would be greatly appreciated.

**Comment by azurespheredev** [comment, marked as answer]
Your issue would happened in Next.js 14.x.
You can use `unoptimized` attribute in Image component. Here is my solution.

```
import Image from "next/image";

<Image
  src={imageURL}
  width={48}
  height={48}
  alt="logo"
  unoptimized
/>
```

**Reply by morganm94** [reply]
Thank you for your quick response and that worked right! 😄
